esp/scm/ws_topology.ecm

```diff
@@ -575,6 +576,7 @@ ESPrequest [nil_remove] SystemLogRequest
     string Name;
     string Type;
     int   Zip;
+    [min_ver("1.32")] string CompType;
```

Comment:
This should have some kind of documentation.  Does eclwatch need to be updated to provide this argument, or will it mean that the legacy SystemLog option will no longer work.

Reply:
The CompType is used to set the component variable inside the getConfigurationDirectory(). So far, I haven't found which ConfigurationDirectory needs that variable. I add the CompType here just in case we need it some time later. Do you know when the component variable has to be specified?

Reply:
My conclusion was for the default environment file none of default definitions need it to be set (because it is the trailing part of the file path).  Someone could define a strange environment where all cases will need it to be defined.  (It would also be required if a component-specific override was defined.)
ComponentType would be a better name than CompType, slightly clearer what it corresponds to.

Reply:
If it is not specified then the user will be able to get to any logs.